doctr/models/recognition/sar/pytorch.py

```diff
@@ -50,28 +50,41 @@ class AttentionModule(nn.Module):
 
     def __init__(self, feat_chans: int, state_chans: int, attention_units: int) -> None:
         super().__init__()
-        self.feat_conv = nn.Conv2d(feat_chans, attention_units, 3, padding=1)
+        self.lstm_cell = nn.LSTMCell(feat_chans, state_chans)
+        self.feat_conv = nn.Conv2d(feat_chans, attention_units, kernel_size=3, padding='same')
```

Comment:
string passing to "padding" has been introduced in PyTorch 1.9.0

Considering our current requirements, I'd suggest going back to integer for now :)